### PR TITLE
new records

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/1776

### CODE REVIEW

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: new/LIT6473PPAynaTela.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6473PPAynaTela" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎተ፡
+                    አይነ፡ ጥላ፡ ወአይነ፡ ወርቅ፡ ወአይነ፡ ሥራይ፡</title>
+                <title xml:lang="gez" corresp="#t1">Ṣalota ʾayna ṭǝlā wa-ʾayna warq wa-ʾayna wagi wa-ʾayna śǝrāy</title>
```
Comment (DariaElagina) [inline new/LIT6473PPAynaTela.xml:10]: wa-ʾayna wagi is lacking in fidel. Or vice versa.

File: new/LIT6474PPAynaTela.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6474PPAynaTela" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ ሕማመ፡ አይነ፡ ጥላ፡ ወአይነ፡
+                    ወርቅ፡ ወአይነ፡ አጋንንት፡ ወልሳነ፡ ቡዳ፡ ወቁመኛ፡ ወመርተኛ፡ ወአይነ፡ አጋንት፡ ወሕማመ፡ ፍጥረት፡ </title>
+                <title xml:lang="gez" corresp="#t1">Ṣalot baʾǝnta ḥǝmāma ʾayna ṭǝlā wa-ʾayna warq wa-ʾayna ʾagānǝntt wa-lǝsāna budā wa-qumaññā
```
Comment (DariaElagina) [inline new/LIT6474PPAynaTela.xml:10]: ʾagānǝntt?

File: new/LIT6478PPMasaraAganent.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6478PPMasaraAganent" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ ማዕሠረ፡ አጋንንት፡ ወቁርጥማታ፡ ወውጋተ፡ ወሆድ፡ ቁርጸት፡ ወተሰዋሾ፡</title>
+                <title xml:lang="gez" corresp="#t1">Ṣalot baʾǝnta māʿśara ʾagānǝnt za-qurṭǝmāta wa-wǝgāta wa-hod qurṣat wa-tasawāšo</title>
```
Comment (DariaElagina) [inline new/LIT6478PPMasaraAganent.xml:9]: qurṭǝmātā?

**Comment by thea-m** [comment]
Thank you!

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**